libs/velo-external-db-commons/src/libs/errors.ts

```diff
@@ -120,3 +120,14 @@ export class UnrecognizedError extends BaseHttpError {
         super(`Unrecognized Error: ${message}`)
     }
 }
+
+export class CollectionChangeNotSupportedError extends BaseHttpError {
+    collectionName: string
+    fieldName: string
+
+    constructor(message: string, collectionName?: string, fieldName?: string) {
+        super(message)
+        this.collectionName = collectionName || ''
+        this.fieldName = fieldName || ''
```

Comment:
I think it would be better to use ?? instead to handle null/undefined.
because theoretically || could catch "falsy" values as well (I think there's no real impact here unless someone will call is table - 'undefined' / 'null' / 'false'), but still I think ?? is describes better what you're trying to do

Reply:
No problem. I will change it to all the errors that have it